ClosedXML_Tests/Excel/CalcEngine/StatisticalTests.cs

```diff
@@ -242,6 +242,46 @@ public void Init()
             workbook = SetupWorkbook();
         }
 
+        [Test]
+        public void Fisher()
+        {
+            var ws = workbook.Worksheets.First();
+            double value;
+
+            Assert.That(() => ws.Evaluate(@"=FISHER(D3:D45)"),
```

Comment:
Split into a test for the exceptions and a test for the calculated values, using a parametric test, i.e. use the `TestCase` attribute. See other tests for examples.

Reply:
I changed tests for correct cases use for this [TestCase] attribute. Should we also separate tests for exceptions?